com.nitrokey.nitrokey-app2.yml

```diff
@@ -0,0 +1,43 @@
+app-id: com.nitrokey.nitrokey-app2
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: nitrokeyapp
+build-options: 
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env: 
+   - "CARGO_NET_OFFLINE=true"
+cleanup-commands:
+  - "find ${FLATPAK_DEST}/bin -type f | grep -v nitrokeyapp | xargs rm"
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --share=network
+modules:
+  - shared-modules/libusb/libusb.json
+  - pre-pypi-dependencies.json
+  - rust-pypi-dependencies.json
+  - python3-poetry.json
+  - python3-hidapi.json
+  - app-pypi-dependencies.json
+  - name: nkapp
+    buildsystem: simple
+    build-commands:
+      - patch -p1 < disable_nrf_update.patch
+      - sed -i -e 's|2.1.5|2.2.1|g' meta/${FLATPAK_ID}.metainfo.xml
```

Comment:
Can you make this a patch too or fix this upstream? sed is likely to replace unintended things.

Reply:
will make a patch, this will be fixed with the next release - then the patch can be removed

Reply:
done